linode/resource_linode_nodebalancer_node_test.go

```diff
@@ -227,6 +238,18 @@ resource "linode_nodebalancer_node" "foonode" {
 `, label)
 }
 
+func testAccCheckLinodeNodeBalancerNodeInstanceUpdates(label string) string {
+	return testAccCheckLinodeInstanceConfigPrivateNetworking(label, publicKeyMaterial+"_new") + testAccCheckLinodeNodeBalancerConfigBasic(label) + fmt.Sprintf(`
```

Comment:
I'd rather the label change or some other property instead of making the public key invalid

Reply:
Ah yeah, this was mainly to reproduce the original issue (it's an easy way to force recreation). Alternatively we could change the image, region, root pass, etc.